ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1350,238 +1231,6 @@
-          builders:
-            - builder-conformance-kind:
-               conformance_type: 'conformance'
-               ip_version: 'dual'
-               kind_cluster_name: '{test_name}'
-          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-ipv6-(conformance|all).*
-          white_list_target_branches: []
-          allow_whitelist_orgs_as_admins: true
-          admin_list: '{antrea_admin_list}'
-          org_list: '{antrea_org_list}'
-          white_list: '{antrea_white_list}'
-          only_trigger_phrase: true
-          trigger_permit_all: true
-          status_context: jenkins-kind-ipv6-ds-conformance
-          status_url: --none--
-          success_status: Build finished.
-          failure_status: Failed. Add comment /test-kind-ipv6-conformance to re-trigger.
-          error_status: Failed. Add comment /test-kind-ipv6-conformance to re-trigger.
-          triggered_status: null
-          started_status: null
-          wrappers:
-          - timeout:
-              fail: true
-              timeout: 135
-              type: absolute
-          publishers:
-          - archive:
-              allow-empty: true
-              artifacts: '*sonobuoy*.tar.gz'
-              case-sensitive: true
-              default-excludes: true
-              fingerprint: false
-              only-if-success: false
-      - '{name}-{test_name}-for-pull-request':
-            test_name: kind-ipv6-ds-e2e
-            node: 'antrea-kind-testbed'
-            description: 'This is the {test_name} test for {name}.'
-            branches:
-            - ${{sha1}}      
-            builders:
-              - builder-e2e-kind:
-                  ip_version: 'dual'
-                  kind_cluster_name: '{test_name}'
-            trigger_phrase: ^(?!Thanks for your PR).*/test-kind-ipv6-(e2e|all).*
-            white_list_target_branches: []
-            allow_whitelist_orgs_as_admins: true
-            admin_list: '{antrea_admin_list}'
-            org_list: '{antrea_org_list}'
-            white_list: '{antrea_white_list}'
-            only_trigger_phrase: true
-            trigger_permit_all: true
-            status_context: jenkins-kind-ipv6-ds-e2e
-            status_url: --none--
-            success_status: Build finished.
-            failure_status: Failed. Add comment /test-kind-ipv6-e2e to re-trigger.
-            error_status: Failed. Add comment /test-kind-ipv6-e2e to re-trigger.
-            triggered_status: null
-            started_status: null
-            wrappers:
-            - timeout:
-                fail: true
-                timeout: 135
-                type: absolute
-            publishers:
-            - archive:
-                allow-empty: true
-                artifacts: antrea-test-logs.tar.gz
-                case-sensitive: true
-                default-excludes: true
-                fingerprint: false
-                only-if-success: false
       - '{name}-{test_name}-for-pull-request':
```

Comment:
Please also migrate ipam job to cloud yaml. We should remove any kind jobs in lab yaml.

Reply:
kind-flexible-ipam testbed has some special configuration, for that i am thinking to create separate PR.